frontend/lib/Providers/CompareEndpointsModel.dart

```diff
@@ -0,0 +1,105 @@
+import 'package:adpv_frontend/Repository/AbstractEndpointRepository.dart';
+import 'package:adpv_frontend/Repository/RestClient.dart';
+import 'package:dio/dio.dart';
+import 'package:flutter/cupertino.dart';
+
+import '../Models/Endpoint.dart';
+import '../Models/EndpointSummary.dart';
+
+class EndpointModel extends ChangeNotifier {
+  Map<String, EndpointSummary> endpointSummaryMap = {};
+  Map<String, Endpoint> endpointsMap = {};
+  Map<String, bool> selectedChips = {};
+  List<String> commonFields = [];
+  List<String> selectedEndpoints = [];
+  AbstractEndpointRepository repository = RestClient(Dio());
+
+  EndpointModel();
+
+  void selectChips(String label, bool value) {
+    selectedChips[label] = value;
+    if (value) {
+      selectedChips[label] = true;
+    } else {
+      selectedChips[label] = false;
+    }
```

Comment:
Czy to tutaj jest potrzebne?

Reply:
tak, nie wiem jak to na ten moment sprytniej zrobić

Reply:
Tylko popatrz na to:
```dart
selectedChips[label] = value;
if (value) {
  selectedChips[label] = true;
} else {
  selectedChips[label] = false;
}
```
Przypisujesz najpierw wartość, a potem robisz jakieś niepotrzebne if'y i jeszcze raz przypisujesz i wychodzi Ci na to samo co w pierwszej linijce. Dla mnie te 4 linijki if'a do wyrzucenia

Reply:
selectedChips[label] = value; 
jest do wywalenia, bez tych ifów nie działa

Reply:
jestem na 99% pewny, że wtedy miałem inaczej chipsy rozwiązane, przed providerem. I tamto przypisanie było do innej struktury. My bad.  Poprawione